src/kaltura-live-plugin.tsx

```diff
@@ -40,6 +40,7 @@ export enum LiveBroadcastStates {
 export enum OverlayItemTypes {
     None = "None",
     Offline = "Offline",
```

Comment:
not sure if this is the best practice with enums - but this is internal use only . We can improve later